### DISCUSSION TITLE
1.40.2

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#24174] [General]
Feel free to discuss the [1.40.2](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.40.2) release here!

**Comment by Birdgage** [comment]
Mumubiz TYZGTH1CH-D1RF The pairing of the device goes perfect, and all functions and parameters are function. But....after a reboot of Zigbee2MQTT, everything comes online but the on/off function gives an error and will not work, all the other data points are working, such as temp/hum calibration etc , only deleting the device, restarting Zigbee2MQTT and pairing the device will work, until the next reboot, and then again only the on/off function is not working.

**Comment by jan-di** [comment]
I've updated to 1.40.2 due to the support for the detached relay mode of the new SONOFF ZBMINIR2 - it works great and enables me to control smart light bulbs without cutting them from the power every time. 

On the other hand, there are now some false positive error messages in the log, regarding my Aqara SRTS-A01 devices. See bug report here: https://github.com/Koenkk/zigbee2mqtt/issues/24312

**Comment by purche42** [comment]
Hi Koen,

With update to 1.40.2 I've got problems with switch devices On/Off.  I get status from sensors, lights switches and anythings looks normal also in HA. Devices are visible in Z2M console.  Restarting Z2M solve problem temporarly. With next restart I have the same problem again.

Since I use auto update (watchtower) of docker containers I searched a while. The problem starts with Z2M update from 1.40.1 to 1.40.2 two weeks ago.    
I switched back to 1.40.1 

See also ights problem #24293
